config/application.rb

```diff
@@ -36,3 +36,6 @@ class Application < Rails::Application
     config.active_record.raise_in_transactional_callbacks = true
   end
 end
+
+puts "Rails.env: #{Rails.env}"
+puts "database: #{MadelineSystem::Application.config.database_configuration[::Rails.env]['database']}"
```

Comment:
I've found it very useful to have a visual confirmation of the rails env and current database during startup.  I've been burned a few times when I'd been inadvertently operating from an environment or database other than I had realized.